opensrp-eusm/src/main/java/org/smartregister/eusm/fragment/EusmTasksRegisterFragment.java

```diff
@@ -130,6 +130,8 @@ public void onViewClicked(View view) {
                         presenter.startForm(structureDetail, taskDetail, getBeneficiaryConsultationForm());
                     } else if (AppConstants.NonProductTasks.WAREHOUSE_CHECK.equalsIgnoreCase(taskDetail.getEntityName().trim())) {
                         presenter.startForm(structureDetail, taskDetail, getWarehouseCheckForm());
+                    } else if (AppConstants.TaskCode.FIX_PROBLEM_CONSULT_BENEFICIARIES.equalsIgnoreCase(taskDetail.getTaskCode())) {
```

Comment:
Whats the value of `taskDetail.getEntityName()` here? If its the same as taskCode kindly use the entityName.